README.adoc

```diff
@@ -247,3 +247,13 @@ If you need any permissions also in a namespace in host cluster (to be used main
     - namespace: <namespace-name>
     ...
 ```
+
+=== Running `ksctl` locally
```

Comment:
minor - maybe it can be a sub-section of https://github.com/kubesaw/ksctl?tab=readme-ov-file#generate-ksctlyaml-files , something like `Testing the ksctl generate cli-configs command locally`

Reply:
+1

Reply:
Thanks so much for the suggestion, @mfrancisc! Goingggg